src/js/big-play-button.js

```diff
@@ -59,7 +59,7 @@ class BigPlayButton extends Button {
 
     const playFocus = () => playToggle.focus();
 
-    if (isPromise(playPromise)) {
+    if (playPromise && isPromise(playPromise)) {
```

Comment:
`isPromise` should be doing a null check as well, no, this shouldn't be strictly necessary.

Reply:
I did notice that `isPromise` was checking for `undefined` specifically. I thought that was for a specific reason, but maybe I was wrong?

Reply:
if `play()` is called and no promise was returned, we end up with `undefined`. Though, we'll never go into this if statement if `playPromise` doesn't duck-type into a promise.
I'm ok with leaving it if you want, but it shouldn't be necessary.

Reply:
Ah, while I was testing, if the play was terminated then I saw an error coming from this `if` statement. Perhaps the right way is to return something other than `null` in the cases where a middleware terminates.

Reply:
Ah, I see. Maybe we should just loosen the check in `isPromise`